templates/controllers/tab/settings/productionStage/form/productionStageForm.tpl

```diff
@@ -46,4 +44,3 @@
 		{fbvFormButtons id="productionStageFormSubmit" submitText="common.save" hideCancel=true}
 	{/if}
 </form>
-<p><span class="formRequired">{translate key="common.requiredField"}</span></p>
```

Comment:
Do you remember what it might be? Alec thought no, and nothing's marked that way. Here's a screenshot of what I sent to Alec.

![settings-workflow-production-required](https://cloud.githubusercontent.com/assets/2306629/11783643/322e7ff8-a270-11e5-83a3-5af972b73ff9.png)